README.md

```diff
@@ -45,6 +46,30 @@ If your IDE supports it, you can directly run the .feature files within `/featur
 Run the tests by running `behave` in a command prompt or terminal window.
 * This will run the tests and print the results to console
 
+```
+behave -D product=EPS-FHIR -D env=INT -f behave_cucumber_formatter:PrettyCucumberJSONFormatter -o reports/cucumber_json.json -f
+allure_behave.formatter:AllureFormatter -o allure-results -f pretty features --no-capture --no-capture-stderr --no-skipped --expand --logging-level=DEBUG --tags eps_fhir
+```
+
+change the `env` variable accordingly to either `INT` or `INTERNAL-DEV`
```

Comment:
Does this need a definitive list of supported envs and products for reference?
Also, 'If you *wish* to test...' and 'PFP-APIGEE'?

Reply:
As the development process continues and more capability is added we could enforce this suggestion.
At the moment the current will suffice I think.

Reply:
The typos could still do with fixing.

Reply:
I just noticed the typo thank you.